tests/php/src/ValidatedUrlStylesheetDataGarbageCollectionTest.php

```diff
@@ -5,6 +5,7 @@
  * @package AmpProject\AmpWP
  */
 
```

Comment:
The namespace is missing along with a few classes to be imported. Suggested patch:

```diff
diff --git a/tests/php/src/ValidatedUrlStylesheetDataGarbageCollectionTest.php b/tests/php/src/ValidatedUrlStylesheetDataGarbageCollectionTest.php
--- a/tests/php/src/ValidatedUrlStylesheetDataGarbageCollectionTest.php	(revision cd4dd777102b5f30b32728e0c40cfaecc93b4097)
+++ b/tests/php/src/ValidatedUrlStylesheetDataGarbageCollectionTest.php	(date 1606295527974)
@@ -5,8 +5,12 @@
  * @package AmpProject\AmpWP
  */
 
+namespace AmpProject\AmpWP\Tests;
+
+use AMP_Validated_URL_Post_Type;
 use AmpProject\AmpWP\BackgroundTask\BackgroundTaskDeactivator;
 use AmpProject\AmpWP\BackgroundTask\ValidatedUrlStylesheetDataGarbageCollection;
+use WP_UnitTestCase;
 
 /** @coversDefaultClass \AmpProject\AmpWP\BackgroundTask\ValidatedUrlStylesheetDataGarbageCollection */
 class ValidatedUrlStylesheetDataGarbageCollectionTest extends WP_UnitTestCase {
```

Reply:
Done in 28581d774.